scripts/ci/podman-test.sh

```diff
@@ -11,6 +11,22 @@ make install
 
 criu --version
 
+# Install crun build dependencies
```

Comment:
Please add a comment that we install it from source to test libcriu

Reply:
Thanks, I've updated the comment.